src/main/java/de/mari_023/fabric/ae2wtlib/terminal/WTGuiObject.java

```diff
@@ -67,6 +69,12 @@ public WTGuiObject(final IWirelessTermHandler wh, final ItemStack is, final Play
         }
     }
 
+    public abstract boolean open(PlayerEntity player, ContainerLocator locator);
+
+    public abstract ScreenHandlerType<?> getType();
```

Comment:
Actually we might be able to drop getType() entirely
it is just used by the WirelessCraftConfirmContainer to check if it's null, which it can't be, if the icon isn't empty

Reply:
my bad, it is also used to get the the registry id to go back